package.json

```diff
@@ -76,7 +76,7 @@
     "start": "node tools/proxy/proxy.js",
     "build": "node --max-old-space-size=4096 node_modules/.bin/webpack --env production",
     "@comment build-backend": "Only check for errors in the backend code like done by the CI. This command is not needed to run WEBKNOSSOS",
-    "build-backend": "yarn build-wk-backend && yarn build-wk-datastore && yarn build-wk-tracingstore",
+    "build-backend": "yarn build-wk-backend && yarn build-wk-datastore && yarn build-wk-tracingstore && rm webknossos-tracingstore/conf/messages webknossos-datastore/conf/messages",
```

Comment:
why are these removed?

Reply:
These message files are generated by the script as the tracingstore and datastore are built. When they are not deleted and one tries to run the script again the compilation of the two components somehow fails. After running into this multiple times (me having to manually delete these files) as well as taking care of not pushing these files, I simply wanted the script to also take care of cleaning up the auto-generated message files :)